demo/.browserslistrc

```diff
@@ -0,0 +1,12 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
+
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
+not IE 9-11 # For IE 9-11 support, remove 'not'.
```

Comment:
As we discussed IE11 support should be possible from the angular perspective. Shouldn't this be the same `.browserslistrc` like in frontend-som?

Reply:
it is just a demo and I kept the one from `ng new` but now I put the one from f-som 👍